testing/python/fixtures.py

```diff
@@ -3620,6 +3620,24 @@ def test_fixture_disallow_twice(self):
             def foo():
                 raise NotImplementedError()
 
+    def test_fixture_disallow_on_marked_functions(self):
```

Comment:
We usually put deprecation tests in `deprecated_test.py`, so we ensure that we are also displaying the warnings properly to the user (including that the warning points to the user's code). You can keep those two tests here if you want too.

Reply:
>  (including that the warning points to the user's code)

how do I check that here?

Reply:
We usually use `pytester` to run a separate file, and then check the output `result.stdout.fnmatch_lines`.

Reply:
Did you try this? I think it is important that we are sure that we are pointing to the mark or fixture definition, otherwise it will not be clear for the user which fixture they should fix.